src/core/ProgressBar/index.ts

```diff
@@ -0,0 +1 @@
+export * from './BaseProgressBar';
```

Comment:
There is an empty `base.ts` and `index.ts` is exporting the empty base file which is not required at all.

Reply:
@S-ayanide I'm exporting BaseProgressBar folder to index.ts

Reply:
You won't require a base folder for one single component, it's basically to share the base component with similar component in that chain. You can directly export in index.ts of Progress Bar